lib/detail_rich_text.dart

```diff
@@ -0,0 +1,37 @@
+import 'ui_constants.dart';
+import 'package:flutter/material.dart';
+
+class DetailRichText extends StatelessWidget {
+  const DetailRichText({super.key, required this.subtitle, required this.text});
```

Comment:
Trailing comma, and please run the auto-format in this file

Reply:
done